cordapp-example/workflows-kotlin/src/main/kotlin/com/example/flow/PaybackFlow.kt

```diff
@@ -87,17 +101,26 @@ object PaybackFlow {
             val inputStateAndRef = serviceHub.toStateAndRef<IOUState>(results.states.single().ref)
             val iouState = inputStateAndRef.state.data
 
```

Comment:
`findMoney` should take an `amount` parameter; so it will query the vault for Cash states and return a list, the sum of quantities of Cash states in that list should be greater or equal to the requested amount.
Your current implementation just queries and returns whatever Cash state owned by `ourIdentity` it finds, what if I owned 5 Cash states and I returned one of them where its amount is 1 while the IOU amount is 5, does that mean I can payback an IOU of value 5 with a Cash of value 1?

Reply:
I created simple class `MoneyFinder`. Because I create cash each time I borrow it, I will always have exact amount "waiting" for me to payback.
I know it's not how it should work in relal live, but I simplified it, so I can move forward with it.